typescript/README.md

```diff
@@ -0,0 +1,5 @@
+**Note**: The types provided with rdkit-js are manually maintained, but are intended to match and map all of its functionalities as much as possible. Please log any issue or bug if you encounter problems while using them.
+
+# Example usage
+
+...to be completed...
```

Comment:
@adam-of-barot , your last todo would be to complete this before we merge this PR